R/nested_ifelse_linter.R

```diff
@@ -1,15 +1,15 @@
-#' Block usage of nested ifelse() calls
+#' Block usage of nested `ifelse()` calls
 #'
-#' Calling `ifelse` in nested calls is problematic for two main reasons:
+#' Calling [ifelse()] in nested calls is problematic for two main reasons:
 #'   1. It can be hard to read -- mapping the code to the expected output
 #'      for such code can be a messy task/require a lot of mental bandwidth,
 #'      especially for code that nests more than once
-#'   2. It is inefficient -- `ifelse` can evaluate _all_ of its arguments at
-#'      both yes and no (see https://stackoverflow.com/q/16275149); this issue
+#'   2. It is inefficient -- [ifelse()] can evaluate _all_ of its arguments at
```

Comment:
I think only linking the first usage in a section is the more standard practice... `` `ifelse()` `` instead?

Reply:
Done.